backend/src/ee/services/saml-config/saml-config-service.ts

```diff
@@ -342,7 +348,7 @@ export const samlConfigServiceFactory = ({
             email,
             firstName,
             lastName,
-            authMethods: [authProvider]
+            authMethods: [AuthMethod.EMAIL]
```

Comment:
No. Previously, we updated a field on users to specify that their authentication method is now SAML but this is incorrect because SAML authentication should be bound at the organization-level instead of the user-level. So, users can now retain their personal login methods like Email / Google / GitHub / GitLab whilst being SAML-enforced for specific organizations that have that enforcement.

You'll notice that in this PR we change how we deal with SAML auth validation in the underlying data structure. This adds flexibility such that users can use regular email/password to log into non-org level auth enforced organizations and be required to login with SAML for those that have that enforcement.

Reply:
Curious, what happens if I am apart of a org that has SAML sso configured? Will i just not see those orgs or will it prompt me to login when i switch to that org?